playbooks/deploy-ocp-hybrid-multinode.yml

```diff
@@ -371,3 +413,28 @@
       vars:
         pull_secret: "{{ pull_secret_raw }}"
         trusted_registries: "{{ additional_trusted_registries }}"
+
+# Cleanup play - ensures virtual media ejection on playbook completion or failure
+- name: Cleanup Virtual Media (Always Execute)
+  hosts: nodes
+  gather_facts: false
+  any_errors_fatal: false
+  tags:
+    - always
+    - cleanup
+  tasks:
+    - name: Eject virtual media from all nodes
+      ansible.builtin.include_role:
+        name: "redhatci.ocp.vendors.{{ hostvars[inventory_hostname]['vendor'] | lower }}"
+        tasks_from: eject.yml
+        apply:
+          delegate_to: bastion
+          failed_when: false
+      vars:
+        target_host: "{{ inventory_hostname }}"
+        boot_iso_url: "http://{{ hostvars['bastion']['ansible_default_ipv4']['address'] }}:{{ share_http_iso_port }}/{{ agent_iso_name }}"
+      when: inventory_hostname in groups['masters'] or inventory_hostname in groups['workers']
```

Comment:
Should we add this here as well? 
`- hostvars[inventory_hostname]['requires_cleanup'] | default(false) `